py/server/deephaven/table_listener.py

```diff
@@ -532,6 +492,7 @@ def start(self, do_replay: bool = False) -> None:
             raise RuntimeError("Attempting to start an already started merged listener..")
 
         try:
+            # self.tables[0] is guaranteed to be a refreshing table
```

Comment:
You could actually make this more elegant and future-proof if you:
1. Exposed `is_refreshing` and `update_graph` as properties on your `MergedListener` wrapper (`MergedListenerHandle`). The former is presumably always true, and the latter can delegate to the wrapper Java Object's `getUpdateGraph()`.
2. Used self as the object to pass to `auto_locking_ctx`.

Reply:
That's probably more elegant code but I am OK with the current code as it is more 'explicit'.